app/pages/Auction/index.js

```diff
@@ -106,6 +107,17 @@ export default class Auction extends Component {
     analytics.screenView('Auction');
   }
 
+  async componentDidUpdate(prevProps, prevState) {
+    if (this.props.chain.height !== prevProps.chain.height) {
+      this.refreshInfo();
+    }
+  }
+
+  refreshInfo = throttle(async () => {
+    await this.props.getNameInfo(this.getDomain());
+    await this.props.fetchPendingTransactions();
+  }, 10*1000, {leading: true}) // 10 seconds
```

Comment:
is this throttle rate based on some timeout defined elsewhere, or just an arbitrary rate?

does throttle handle the situation where the async function runs longer than 10 seconds? i.e. if it took 60 seconds to complete, would it fire up to 6-ish concurrent attempts, or would it wait for the first one to resolve before allowing the second to begin?

also, if i'm reading correctly, given the componentDidUpdate is async, and this function awaits on each line, is the async inside of throttle actually providing any benefit?

Reply:
Good points. 10 seconds just seemed enough, since `fetchPendingTransactions()` is a local call and `getNameInfo` is small json and fixed in size (if local or even if remote over network).
I still don't like the idea of random numbers like this, but without a throttle, it will refresh every time the height changes (which can happen more than once a second when syncing). Open to other cleaner ideas.

>given the componentDidUpdate is async, and this function awaits on each line

You're right, we don't really care when it finishes and 118 doesn't need 117 to finish first, so removed await (and async) so they can run in parallel.